bdsbilling/src/main/java/com/appcoins/wallet/bdsbilling/WalletService.kt

```diff
@@ -5,7 +5,10 @@ import io.reactivex.Single
 
 interface WalletService {
 
-    fun getWalletAddress(): Single<String>
+  fun getWalletAddress(): Single<String>
 
-    fun signContent(content: String): Single<String>
+  @Deprecated("Use signContent() instead to avoid duplicated code")
+  fun signContent(content: String): Single<String>
+
+  fun signContent(): Single<WalletAddressModel>
```

Comment:
shouldn't this have a different name? It doesn't sign a content, it signs the wallet address, right?